RFS/src/main/java/com/rfs/common/http/ConnectionContext.java

```diff
@@ -102,6 +124,14 @@ public static class TargetArgs implements Params {
             "--target-password" }, description = "Optional.  The target password; if not provided, will assume no auth on target", required = false)
         public String password = null;
 
+        @Parameter(names = {
+            "--target-sigv4-region" }, description = "The source sigv4 region, if not provided will assume no sigv4 on source", required = false)
```

Comment:
Rename to Target AWS Region or something similar, while this is used for sigv4 signing, its about the region the endpoint is inside.

Reply:
Do we want to require this parameter only if Sigv4 is enabled?